heltour/tournament/admin.py

```diff
@@ -661,7 +661,9 @@ def player_info_view(self, request, object_id, player_name):
             'league': season.league,
             'player': season_player.player,
             'reg': reg,
-            'has_played_20_games': has_played_20_games
+            'has_played_20_games': has_played_20_games,
+            'date_created': str(sp.registration.date_created or sp.player.date_created),
+            'friends': sp.registration.friends or None
```

Comment:
if registration is None both of these lines will fail. I would suggest using this format:

`sp.registration.friends if sp.registration else None`
and
`(sp.registration.date_created if sp.registration else sp.date_created).isoformat()`\

This also uses the `isoformat()`